qt/scientific_interfaces/Indirect/IndirectFitAnalysisTab.cpp

```diff
@@ -290,7 +150,7 @@ void IndirectFitAnalysisTab::setOutputOptionsView(
 }
 
 void IndirectFitAnalysisTab::setFitPropertyBrowser(
-    MantidWidgets::IndirectFitPropertyBrowser *browser) {
+    IndirectFitPropertyBrowser *browser) {
```

Comment:
This function is sharing ownership of the pointer here so I think it makes sense to pass that in? This is a raw pointer rather than smart pointer here because of Qt's way of handling pointer ownership